tests/scripts/test_output.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(realpath "$0")")/config.source"
+
+cleanup() {
+  pulp file repository destroy --name "cli_test_file_repo" || true
+}
+trap cleanup EXIT
+
+expect_succ pulp file repository create --name "cli_test_file_repo" --description "Test repository for CLI tests"
```

Comment:
Is this handled in a way that $OUTPUT looks like it was on a TTY?

Reply:
Nope, let me try something different.

Reply:
So you found something?
I'd be happy to take it without a test, because faking a user session is hard...

Reply:
I did. Check it out.